bip-0003.md

```diff
@@ -87,7 +87,8 @@ No formal or informal decision body governs Bitcoin development or decides adopt
 
 Authors may choose to submit BIPs in MediaWiki or Markdown[^markdown] format.
 
-Each BIP must have a _Preamble_, an _Abstract_, a _Copyright_, and a _Motivation_ section. Authors should consider all issues in the
+Each BIP must have a _Preamble_, an _Abstract_, a _Motivation_, a _Specification, and a _Copyright_ section.
```

Comment:
We have Informational BIPs that don’t have a Specification section, and it seems to me that Informational BIPs do not require a Specification section. The **BIP Types** section specifies that Specification BIPs require a Specification, and heavily implies that Process BIPs do, too (“Process BIPs are like Specification BIPs, …”).